src/drivers/bitbucket_cloud.js

```diff
@@ -254,9 +258,12 @@ class BitbucketCloud {
 
     if (response.status > 300) {
       try {
-        const {
-          error: { message }
-        } = await response.json();
+        const json = await response.json();
+        winston.debug(json);
+        // We have observed BitBucket sometimes sends { "error": { "message": "Some message" } }
+        // and sometimes { "error": "Some message" }. Attempt to get additional context. Fall back
+        // to the empty string to ensure we don't print `undefined` to the user.
+        const message = json.error.message || json.error || '';
         throw new Error(message);
```

Comment:
```suggestion
        throw new Error(json.error.message || json.error || '');
```
Would it be better to merge theste two lines into a single one?

Reply:
Thanks for the suggestion! Agreed & updated :)